src/woo_publications/config/models.py

```diff
@@ -52,6 +52,16 @@ class GlobalConfiguration(SingletonModel):
         blank=False,
         related_name="+",
     )
+    gpp_app_publication_url_template = models.URLField(
+        _("GPP-app publication URL template"),
+        max_length=500,
+        default="",
+        blank=True,
+        help_text=_(
+            "URL pattern to a publication in the GPP app. The special token <UUID> "
+            "will be replaced with the system identifier of each publication."
+        ),
+    )
```

Comment:
If I get the ticket correctly the wanted behavior is that it's either configured or left blank. But as of now we don't validate if the placeholder `<UUID>` is present in the url. Isn't this something we should want? Or is there an actual use case where we want the url to be constructed without the uuid present in it?